### DISCUSSION TITLE
BentoML frontend

### DISCUSSION THREAD

**Discussion by pvbhanuteja** [bentoml/BentoML#4213] [General] [answered]
BentoML frontend looks amazing is there a opensource version of this anywhere >

**Comment by aarnphm** [comment, marked as answer]
I'm not sure if I understand what you are looking for, but you can build any frontend setup to interact with BentoML service.

Otherwise when you access `localhost:<port:3000>`, then it is just OpenAPI frontend.

**Comment by parano** [comment]
The current frontend UI is based on OpenAPI UI. However our team is building a new version of the UI which got much improved design specifically for AI/ML services. Stay tuned!
